Gruntfile.js

```diff
@@ -245,11 +197,11 @@ module.exports = function(grunt) {
 
   // > grunt build
   // Build for production.
-  grunt.registerTask('build', ['clean:dist', 'copy:assets', 'sass:prod', 'postcss:prod', 'webpack:prod', 'modernizr:all']);
+  grunt.registerTask('build', ['clean:dist', 'copy:assets', 'sass:prod', 'postcss:prod', 'webpack', 'modernizr:all']);
 
   // > grunt build:debug
   // Build for development.
-  grunt.registerTask('build:debug', ['clean:dist', 'copy:assets', 'sass:debug', 'postcss:debug', 'webpack:debug', 'modernizr:all']);
+  grunt.registerTask('build:debug', ['clean:dist', 'copy:assets', 'sass:debug', 'postcss:debug', 'webpack', 'modernizr:all']);
```

Comment:
Yep, the environment switching is handled by `NODE_ENV` in the Webpack config. This was a bit of a stop-gap on the way to completely pulling this stuff out.